docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst

```diff
@@ -125,55 +143,34 @@ This cause's restrictions have not changed since GBD 2017.
-cause-deleted all-cause mortality rate, should be zero.
-
-Model Assumptions and Limitations
-+++++++++++++++++++++++++++++++++
-
-1. This cause model assumes no recovery from MM and RRMM since myeloma is an 
-   incurable disease. Patients with MM will inevitably develop relapse and the 
-   health outcomes worsen with every relapse and line of treatment.
-2. This cause model assumes that the GBD incidence rate corresponding to the incidence 
-   of symptomatic MM. That's said, we are comfortable using GBD incidence of MM 
-   as the detection rate of symptomatic MM cases. The incidence of RRMM will be 
-   calculated from survival regression analysis using Cox's proportional hazard model.
-3. The asymptomatic/idolent state (smoldering MM) is excluded from this cause 
-   model because we are not interested in the screening and early managment for 
-   MM. As a result, the simulation will not track/model simulants with asymptomatic 
-   condition.
-4. YLLs are substantially larger than YLDs for this cause. For now, we will not 
-   build a disability component to capture those secondary outcomes.
-5. Based on available data, the most advanced disease state in cause model is 
-   fourth or higher relapse of RRMM. We intended to track deaths from simulants 
-   who have developed fourth relapse and received fifth-line of treatment but ignore 
-   the incident cases from fourth relapse to higher relapse of RRMM. As a result, 
-   we will not calculate progress-free survival among simulants with fourth or 
-   higher relapse of RRMM.
-
-
 Cause Model Diagram
 +++++++++++++++++++
 
 .. image:: cause_model_diagram.svg
 
+Overview
+++++++++
+
+To study the impact of different lines of treament for myeloma patients, we
+split multiple myeloma into two disease states: newly-diagnosed multiple myeloma (NDMM);
+and relapsed and/or refractory multiple myeloma (RRMM). The RRMM state consists of
+multiple relapse stages. This MM cause model is intended to simulate MM incidence,
+relapse, and mortality. The inputs for this cause model come from GBD 2019 estimates,
+scientific literature, and survival analysis supported by Flatiron data.
+
+In our model, treatment and relapse perfectly correspond; that is,
+a simulant in the NDMM state is receiving Line 1 treatment, a simulant in the MM_first_relapse state
+is receiving Line 2 treatment, etc. New treatment assignment is therefore performed only at the time of
+relapse.
+
+Notably, the OS and TTD survival analyses supported by Flatiron data provide
+data on death *from any cause* among multiple myeloma patients and
+not death due to multiple myeloma specifically.
+Therefore, the excess mortality defined in this document will be the only source
+of mortality among simulants with multiple myeloma and excess mortality among
+simulants with multiple myeloma for "other causes," as typically defined by the
+cause-deleted all-cause mortality rate, should be zero.
```

Comment:
I tried to explain previously why I want additional background mortality here, but in the spirit of keeping our changes minimal, I will not push more on it now.

Reply:
To briefly summarize the approach I would prefer, though it is a stretch goal and I don't want to distract from more important things:

Our curves, instead of depending only on line number and ASCT, should also depend on other covariates that we think affect hazard of death, primarily age. That way, we can account for the variation in death patterns that we would normally be getting from GBD (by age and sex) without double-counting any deaths (our "EMR" for this cause is derived from data on all-cause mortality), **and** allow MM to have age-varying EMR.

If we added the GBD other-causes mortality on top of our age-invariant EMR, we would be double-counting all non-MM deaths that occur among people with MM, and still not allowing MM to be more fatal in older people.

Perhaps the ideal would be to use GBD EMR for this cause and essentially rake the EMR of all our sub-MM states to that envelope, but that would be complicated.